### PR TITLE
Review16 07 2020

### PR DESCRIPTION
**Pull request by KaHBrat**

### CODE REVIEW

File: TrustCenterSearch.Presentation/ViewModel.cs [hunk 1]
```diff
@@ -182,45 +173,43 @@ private void RemoveTrustCenterFromFilterCommandExecute(TrustCenterMetaInfo trust
             this.CertificatesCollectionView.Refresh();
         }
 
+        #endregion
+
+        #region Methods
+
+        private void GetTrustCenterHistory()
+        {
+            foreach (var trustCenterHistoryName in Core.GetTrustCenterHistory())
```
Comment (TobiasMeissner2000) [inline TrustCenterSearch.Presentation/ViewModel.cs:181]: this.

File: TrustCenterSearch.Presentation/ViewModel.cs [hunk 2]
```diff
@@ -182,45 +173,43 @@ private void RemoveTrustCenterFromFilterCommandExecute(TrustCenterMetaInfo trust
...
+                TrustCenterHistoryActive.Add(trustCenterHistoryName);
```
Comment (TobiasMeissner2000) [inline TrustCenterSearch.Presentation/ViewModel.cs]: this.

File: TrustCenterSearch.Presentation/ViewModel.cs [hunk 3]
```diff
@@ -182,45 +173,43 @@ private void RemoveTrustCenterFromFilterCommandExecute(TrustCenterMetaInfo trust
...
-            return false;
+                entry.Issuer.ToLower(),
+                entry.Subject.ToLower(),
+                entry.SerialNumber.ToLower(),
+                entry.NotBefore.ToLower(),
+                entry.NotAfter.ToLower(),
+                entry.Thumbprint.ToLower()
+            };
+
+            return certificateAttributes.Any(atr => atr.Contains(SearchBarInput.ToLower()));
```
Comment (TobiasMeissner2000) [inline TrustCenterSearch.Presentation/ViewModel.cs]: this.

**Review by TobiasMeissner2000 — CHANGES_REQUESTED** [review]
three missing this.

**Comment by KaHBrat** [comment]
added the missing "this"

**Review by TobiasMeissner2000 — APPROVED** [review]

**Pull request merged**